apps/src/templates/instructions/HeightResizer.jsx

```diff
@@ -46,25 +46,58 @@ class HeightResizer extends React.Component {
     dragStart: 0
   };
 
+  componentDidMount() {
+    this.resizerRef.addEventListener('mousedown', this.onMouseDown, {
+      passive: false
+    });
+    this.resizerRef.addEventListener('mouseup', this.onMouseUp, {
+      passive: false
+    });
+    this.resizerRef.addEventListener('mousemove', this.onMouseMove, {
+      passive: false
+    });
+    this.resizerRef.addEventListener('touchstart', this.onMouseDown, {
+      passive: false
+    });
+    this.resizerRef.addEventListener('touchend', this.onMouseUp, {
+      passive: false
+    });
+    this.resizerRef.addEventListener('touchmove', this.onMouseMove, {
+      passive: false
+    });
+  }
+
+  componentWillUnmount() {
+    this.resizerRef.removeEventListener('mousedown', this.onMouseDown);
+    this.resizerRef.removeEventListener('mouseup', this.onMouseUp);
+    this.resizerRef.removeEventListener('mousemove', this.onMouseMove);
+    this.resizerRef.removeEventListener('touchstart', this.onMouseDown);
+    this.resizerRef.removeEventListener('touchend', this.onMouseUp);
+    this.resizerRef.removeEventListener('touchmove', this.onMouseMove);
+  }
+
   componentDidUpdate(_, state) {
     // Update listeners as dragging state changes
     if (this.state.dragging && !state.dragging) {
-      document.addEventListener('mousemove', this.onMouseMove);
-      document.addEventListener('mouseup', this.onMouseUp);
+      document.addEventListener('mousemove', this.onMouseMove, {
+        passive: false
+      });
+      document.addEventListener('mouseup', this.onMouseUp, {passive: false});
```

Comment:
is there a reason we  aren't also adding touch listeners here?

Reply:
It's a great question, and I'm not yet sure.  But in my testing, they are needed for mouse dragging, but not for iOS Safari touch dragging.